### ISSUE TITLE
Update to version 0.25

### ISSUE THREAD

**Issue by marta-platoniq** [Platoniq/decidim-module-decidim_awesome#137]
Update the module to the last version of Decidim (0.25)

**Comment by larsUE** [comment]
Hey guys! Do you already have a time horizon for this? Has anybody already tested the compatability with upgrade and is there anything we might be able to support you with? Sorry for the many questions ;)

**Comment by furilo** [comment]
We'd like to contribute to this. Are you already working on this?

**Comment by microstudi** [comment]
Otherwise, we will probably work on this on January

**Comment by entantoencuanto** [comment]
Hi, I've take a look to the changes required to upgrade to 0.25 and one of the most delicate points is related with the editor images functionality. This part has been included in Decidim (decidim/decidim#8250) with some changes to use ActiveStorage instead of CarrierWave. The idea would be:

* Add a migration to change some copied decidim_awesome_editor_images index names (my fault :facepalm:) to the decidim_editor_images table to avoid conflicts with Decidim.
* Add a task to move existing Decidim Awesome editor images to Decidim editor images and migrate them from CarrierWave to ActiveStorage
* Remove the code of the feature replicated in Decidim. In future releases a migration to remove current decidim_awesome_editor_images table could be added.

Do you agree with these changes?

**Comment by microstudi** [comment]
Yeah, I created this table in order to keep track of uploaded images via awesome. Never really used for anything yet. It seems that this type of behaviour is already covered by ActiveStorage so I view your proposal perfectly valid.

Would you start working on this @entantoencuanto?

**Comment by entantoencuanto** [comment]
Yes, my idea is to get it done sometime in the next two weeks

**Comment by microstudi** [comment]
Great, and thanks!
Let me know if you need anything

**Comment by paarals** [comment]
hi all! our plan is to finish it this week, or at most the next. a hug and thank you all for contributing! :-)
